QueAns/admin.py

```diff
@@ -0,0 +1,76 @@
+from django.contrib import admin
+from .models import Question, Meetings, Message, Voting, Satisfaction
+# from datetime import date
+
+
+class MeetingsAdmin(admin.ModelAdmin):
+    list_display = ['meetings_text']
+
+    def get_readonly_fields(self, request, obj=None):
+        if request.user == obj.moderator:
+            return ['moderator']
+        else:
+            return ['id','meetings_text', 'venue', 'moderator', 'start_time', 'end_time']
```

Comment:
- Also, shouldn't all field be editable for a superuser?
- This means that once I have added a meeting, anyone can't change it.

Reply:
Only the moderator of the meeting is allowed to change the meeting. I can add the supersuer if you want?